examples/uefi/libvirt.tf

```diff
@@ -0,0 +1,43 @@
+provider "libvirt" {
+    uri = "qemu:///system"
+}
+
+resource "libvirt_network" "dev-cluster" {
+   name = "dev-cluster"
+   domain = "dev-cluster.local"
+   mode = "nat"
+   addresses = ["10.0.200.0/24"]
+}
```

Comment:
First of all thanks for having provided an example, this is really appreciated! I would prefer to remove the parts of the `.tf` file that are not related with the main topic of this example: uefi and nvram.

Could you remove the custom network please?